server/templates/registration/login.html

```diff
@@ -1,34 +0,0 @@
-{% extends "server/template.html" %}
-
-{% block body %}
-
-  {% if form.errors %}
-    <p>שם משתמש או סיסמה שגויים. נסה שוב</p>
-  {% endif %}
-
-  {% if next %}
-    {% if user.is_authenticated %}
-      <p>Your account doesn't have access to this page. To proceed,
-      please login with an account that has access.</p>
-    {% else %}
-      <p>התחבר כדי לראות דף זה.</p>
-    {% endif %}
-  {% endif %}
-
-  <form method="post" action="{% url 'login' %}">
-    {% csrf_token %}
-      <div class="form-group">
-        <label class="control-label  ">שם משתמש</label>
-        {{ form.username }}
-      </div>
-
-      <div class="form-group">
-        <label class="control-label  ">סיסמא</label>
-        {{ form.password }}
-      </div>
-
-    <input type="submit" value="התחבר" />
-    <input type="hidden" name="next" value="{{ next }}" />
-  </form>
-
-{% endblock %}
```

Comment:
Why'd you delete this file? It seems necessary to me.
When I try to run in your branch and enter /server it tries to redirect me to `http://localhost:8000/accounts/login/?next=/server/` but then throws a `TemplateDoesNotExist at /accounts/login/` error.

Reply:
yup